lamindb_setup/core/_settings_storage.py

```diff
@@ -92,6 +93,14 @@ def init_storage(storage: UPathStr, region: Optional[str] = None) -> "StorageSet
         ssettings._description = f"Created as default storage for instance {uid}"
         ssettings._uuid = init_storage_hub(ssettings)
         logger.important(f"registered storage: {ssettings.root_as_str}")
+    if root.startswith("s3://"):
+        from .upath import create_path
+        from ._settings import settings
+
+        # we need to touch a 0-byte object in the storage location to avoid
+        # permission errors from leveraging s3fs on an empty subdirectory
+        path = create_path(root, settings.user.access_token)
```

Comment:
For the user, this shouldn't be necessary because we already `create_path()` when constructing the `StorageSettings` object. 🤔 

And for the hub, we manage access to storage through global AWS creds anyway, hence, it shouldn't be needed there, too. 🤔

Reply:
I agree, here `ssettings.root` should be used instead of initialization with `create_path`. Also only if `ssettings.is_cloud` and `storage == "create-s3"`

Reply:
Also `settings.user.access_token` should not be passed directly to `create_path`, it is retrieved automatically there.

Reply:
Also i don't actually like that it is done on the client side, because it forces to initialize root at the very beginning. Can't this be done on the hub side via global aws credentials?

Reply:
> For the user, this shouldn't be necessary because we already create_path() when constructing the StorageSettings object.

This makes sense to me. I should just pass `settings.storage.root` to `mark_storage_root`.

> And for the hub, we manage access to storage through global AWS creds anyway, hence, it shouldn't be needed there, too.

This I don't fully understand. What do you mean by hub?

Reply:
> This I don't fully understand. What do you mean by hub?

For the hub test suite you mean, correct?

Reply:
Not settings.storage.root, but `ssettings.root` from the code above.

Reply:
Does anything happen in the hub on triggering `init_storage_hub`?

Reply:
> Does anything happen in the hub on triggering init_storage_hub?

Just a simple upsert to the storage table with fields from `ssetings`.